src/sentry/api/endpoints/sentry_app_features.py

```diff
@@ -14,3 +16,26 @@ def get(self, request, sentry_app):
             paginator_cls=OffsetPaginator,
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def put(self, request, sentry_app):
+
+        current = request.data.get("current")
+        value = request.data.get("value")
+
+        features_dict = dict(Feature.as_choices())
+        if not current or not features_dict[current]:
+            return self.respond(
+                '"current": ["This field is required."]', status=status.HTTP_400_BAD_REQUEST
+            )
+
+        if not value or not features_dict[value]:
```

Comment:
@NisanthanNanthakumar shouldn't you do `features_dict.get(value)` to avoid a key error?